helm_deploy/hmpps-restricted-patients/values.yaml

```diff
@@ -32,6 +33,7 @@ generic-service:
     REDIS_TLS_ENABLED: "true"
     TOKEN_VERIFICATION_ENABLED: "true"
     FRONTEND_COMPONENTS_ENABLED: "false"
+    APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY);IngestionEndpoint=https://northeurope-0.in.applicationinsights.azure.com/;LiveEndpoint=https://northeurope.livediagnostics.monitor.azure.com/"
```

Comment:
we were getting a warning in the logs about this so thought I'd fix at the same time